### PR TITLE
[Enhancement] Add the unhealthy tablet id to schema change job error message

### PR DESCRIPTION
**Pull request by gengjun-git**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

Add the unhealthy table id to the error message. To help troubleshoot problems.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 7 / 9 (77.78%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/catalog/OlapTable.java|4|6|66.67%|[1479, 1495]|
|:large_blue_circle:|com/starrocks/alter/AlterJobV2.java|3|3|100.00%|[]|

**Comment by wanpengfei-git** [comment]
starrocks_be_unittest succeeded.

File: fe/fe-core/src/main/java/com/starrocks/alter/AlterJobV2.java [hunk 1]
```diff
@@ -203,24 +203,24 @@ public final boolean cancel(String errMsg) {
      */
     protected boolean checkTableStable(Database db) throws AlterCancelException {
         OlapTable tbl;
-        boolean isStable;
+        long unHealthyTabletId;
```
Comment (nshangyiming) [inline fe/fe-core/src/main/java/com/starrocks/alter/AlterJobV2.java:206]: need to initialize with -1?

**Review by nshangyiming — DISMISSED** [review]

**Review by nshangyiming — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
starrocks_be_unittest succeeded.

File: fe/fe-core/src/main/java/com/starrocks/alter/AlterJobV2.java [hunk 2]
```diff
@@ -203,24 +203,24 @@ public final boolean cancel(String errMsg) {
      */
     protected boolean checkTableStable(Database db) throws AlterCancelException {
         OlapTable tbl;
-        boolean isStable;
+        long unHealthyTabletId = -1L;
```
Comment (Astralidea) [inline fe/fe-core/src/main/java/com/starrocks/alter/AlterJobV2.java:206]: better to add a constant, use the variable name to mark what -1 means.

**Comment by wanpengfei-git** [comment]
starrocks_be_unittest succeeded.

**Review by nshangyiming — APPROVED** [review]

**Review by Astralidea — APPROVED** [review]

**Comment by gengjun-git** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 7 / 9 (77.78%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/catalog/OlapTable.java|3|5|60.00%|[1479, 1486]|
|:large_blue_circle:|com/starrocks/alter/AlterJobV2.java|4|4|100.00%|[]|

**Review by meegoo — APPROVED** [review]

**Pull request merged**